### PR TITLE
bootstrap: fixes for recent teuthologies on recent openSUSEs

### PR DESCRIPTION
**Pull request by smithfarm**
Signed-off-by: Nathan Cutler <ncutler@suse.com>

### CODE REVIEW

File: bootstrap [hunk 1]
```diff
@@ -14,8 +14,9 @@ fi
 
 case "$(uname -s)" in
 Linux)
-    case "$(lsb_release --id --short)" in
-    Ubuntu|Debian)
+    source /etc/os-release
+    case $ID in
+    debian|ubuntu|devuan)
```
Comment (kshtsk) [inline bootstrap]: devuan => debian ?

**Comment by smithfarm** [comment]
@susebot run teuthology with suse:tier0

**Comment by kshtsk** [comment]
@susebot run teuthology with suse/tier0

**Comment by smithfarm** [comment]
@susebot run teuthology with suse/tier0

**Comment by smithfarm** [comment]
@susebot run teuthology with suse/tier0

**Pull request merged**